addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5654,6 +5654,9 @@ registry.ImageTools = ImageHandlerOption.extend({
      * @override
      */
     _relocateWeightEl() {
+        if (!this.$overlay.data('$optionsSection')) {
+            return;
+        }
```

Comment:
Why thy bug happens when the user goes from Float to Grid and not from Grid to float ? :thinking: